pkg/scheduler/scheduler.go

```diff
@@ -94,15 +94,21 @@ func New(queues *queue.Manager, cache *cache.Cache, cl client.Client, recorder r
 		preemptor:               preemption.New(cl, recorder),
 		admissionRoutineWrapper: routine.DefaultWrapper,
 		waitForPodsReady:        options.waitForPodsReady,
+		leaderElection:          leaderElection,
 	}
 	s.applyAdmission = s.applyAdmissionWithSSA
 	return s
 }
 
-func (s *Scheduler) Start(ctx context.Context) {
+func (s *Scheduler) Start(ctx context.Context) error {
```

Comment:
Add a comment for what this is implementing.

Reply:
done